doc/api/console.md

```diff
@@ -230,6 +230,11 @@ console.timeEnd('100-elements');
 // prints 100-elements: 225.438ms
 ```
 
+*Note: As of node v6.0.0, timeEnd() deletes the timer to avoid leaking it. On
```

Comment:
Can you use `Node.js v6.0.0` here please? 

Maybe also instead of "timeEnd()", do `console.timeEnd()`